fastchat/conversation.py

```diff
@@ -958,6 +958,8 @@ def get_conv_template(name: str) -> Conversation:
     Conversation(
         name="falcon-chat",
         roles=("User", "Falcon"),
+        system_template="System: {system_message}\n",
+        system_message="\nYou are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe.  Your answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure that your responses are socially unbiased and positive in nature.\n\nIf a question does not make any sense, or is not factually coherent, explain why instead of answering something not correct. If you don't know the answer to a question, please don't share false information.",
```

Comment:
What is the source of this system message?

Reply:
It's from https://github.com/philschmid/easyllm/blob/main/notebooks/falcon-180b-chat.ipynb